packages/components/src/templates/next/components/complex/Infopic/Infopic.tsx

```diff
@@ -69,7 +73,9 @@ export const Infopic = ({
           <div className={compoundStyles.button()}>
             <LinkButton
               LinkComponent={LinkComponent}
-              href={getReferenceLinkHref(buttonUrl, site.siteMap)}
+              href={getSanitizedLinkHref(
+                getReferenceLinkHref(buttonUrl, site.siteMap),
+              )}
```

Comment:
since all `getReferenceLinkHref`s use `getSanitizedLinkHref`, should it be built into the function instead

Reply:
Oh ya hor, fixed in 7a87b8606c09bc7f280cb9675120eb57a8b047b1.